flow_prompt/prompt/base_prompt.py

```diff
@@ -109,8 +26,7 @@ def get_params(self):
 
     def add(
         self,
-        content: t.Any,
-        content_type: str = "text",
```

Comment:
why w/o content_type? It's better to have it in the prompt. I find that the fact that we have diff types in the context would be difficult for us. 
We def need to know which content_type it is